src/configs/accessories/invert-all-axes

```diff
@@ -0,0 +1,3 @@
+opt_set INVERT_X_DIR true
```

Comment:
Why mess with this in the firmware? One of the motors is going to be backwards anyway, you might as well reverse the plug.

Reply:
I'll wait and hear what you have to say, but I'd like to merge everything else, but I don't really like this inverting stuff.

Reply:
SKR boards use JST connectors which can not be easily reversed. Given you have correct cables of-course, but dupont does not give a secure connection when pressed into JST.

Granted, one could swap Y1 and Y2 and Z1 and Z2 to reverse the direction of Y and Z, however X is not easily reversed and unlike dupont, where pins are easy to pull out and rearrange, it's a lot more work to dig pins out of a JST connector.

I noticed direction pins are inverted in Klipper's [Example SKR 1.3 config](https://github.com/KevinOConnor/klipper/blob/master/config/generic-bigtreetech-skr-v1.3.cfg). I tested it and steppers do move in the correct direction with TMC2209s. I did not try dropping in DRV8825s to see if this weird behavior where TMC2209s move in a different direction to DRV8825s is some Marlin bug and klipper folks have actually got it right or is it actually some board weirdness.

Reply:
One more detail. This issue does not affect SKR Pro at all. If I drop an SKR Pro board into the same machine with the same non-reversible JST connectors, both TMCs and DRVs move in the same direction with the only config change to firmware config being stepper driver type.

Reply:
Let's move this convesation to #12 to decouple this from the rest of the changes.

I've removed this commit from the updated branch.